example/cuplaReduce/README.md

```diff
@@ -0,0 +1,3 @@
+# About
+
+This example demonstrates how `cupla` can be used together with `vikunja`. The example can only be built alone, because `cupla` and `alpaka` cannot be built in parallel. This is a problem of the CMake of the two libraries.
```

Comment:
> The example can only be built alone, because `cupla` and `alpaka` cannot be built in parallel.

What is the issue here? Is it the dependency chain?

Reply:
The Problem is the global definition of the alpaka acc's. Cupla only allows one active acc but alpaka requires two, if the CUDA backend is enabled. One for the host (cpu) and one for the device (CUDA).

Reply:
Okay. This sounds like a cupla issue we need to fix.

Reply:
I already thought about. I see two possible solutions for it.

1. introduce own CMake arguments. like `CUPLA_ACC_GPU_CUDA_ENABLE`
2. Add a automatic ranking, if more than one acc is enabled. For example, if `ALPAKA_ACC_CPU_B_SEQ_T_SEQ_ENABLE` and `CUPLA_ACC_GPU_CUDA_ENABLE` are enabled, prefer `CUPLA_ACC_GPU_CUDA_ENABLE`

cc @psychocoderHPC